source/utilities.tex

```diff
@@ -63,7 +63,7 @@
 
   // \ref{utility.as.const}, \tcode{as_const}
   template<class T>
-    constexpr add_const_t<T>& as_const(T& t) noexcept;
+    constexpr const T& as_const(T& t) noexcept;
```

Comment:
This changes the mangled name for `std::as_const` specializations, but I don't think we need to care about that.

Reply:
It is a normative change though, so not editorial.

Reply:
Reverted.